lib/wasi/src/state/builder.rs

```diff
@@ -152,6 +171,55 @@ impl WasiStateBuilder {
         self
     }
 
+    /// Adds a container this module inherits from
+    pub fn use_webc<Name>(&mut self, webc: Name) -> &mut Self
```

Comment:
Should it be a full webc package instead of a `Name` ?

Reply:
Yes, in fact it should probably use the same parsing logic Felix implemented for the CLI, or just always require a full WebC specifier (this is why I wanted a parser in `webc`!)

But as above, I'm also moving this to a central location for webc handling.

Reply:
Part of #3603